### DISCUSSION TITLE
default shift+padnum menu shortcut customization

### DISCUSSION THREAD

**Discussion by ecco00** [bangcorrupt/hacktribe#56] [Ideas]
it would be awesome to be able to customize the default shift+numpad 
menu shortcut destination

**Comment by ptowns** [comment]
That would be awesome, especially since they are not labeled on the face I see no reason why not. Put my most used shortcuts in easier to remember places.

**Comment by bangcorrupt** [comment]
I think I can make this work, at least for some of them.

**Reply by ecco00** [reply]
which one of them? would it be possible to assign different MFXs to different pads?

**Reply by bangcorrupt** [reply]
The existing shortcuts select menu nodes without executing the functions.  I think I can change the menu nodes selected without too much difficulty.  If I can make shortcuts that execute functions it will come much later.

**Reply by ecco00** [reply]
awesome!!! AWESOME!!! can't wait

**Comment by ptowns** [comment]
Off subject,  but I'm a big sp404 fan and the new sp404 mk2 let's you change the FX shortcuts. So the button that says filter can be the delay and the delay button can be the filter... because that makes sense.  🤪

**Reply by bangcorrupt** [reply]
Thank you for your support!  :smile:
